cronos/models/job.py

```diff
@@ -1,3 +1,12 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+
+class Job(models.Model):
```

Comment:
Add a class comments to describe what does this model do and cover each individual field